modules/nodes-descheduler-strategies.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-strategies_{context}"]
+= Descheduler strategies
+
+The following descheduler strategies are available:
+
+Low node utilization::
+The `lownodeutilization` strategy finds nodes that are underutilized and evicts Pods, if possible, from other nodes in the hope that recreation of evicted Pods will be scheduled on these underutilized nodes.
++
+The underutilization of nodes is determined by several configurable threshold parameters: CPU, memory, and number of Pods. If a node's usage is below the configured thresholds for all parameters (CPU, memory, and number of Pods), then the node is considered to be underutilized.
++
+You can also set a target threshold for CPU, memory, and number of Pods. If a node's usage is above the configured target thresholds for all parameters, then the node's Pods might be considered for eviction.
++
+Additionally, you can set the strategy to activate only when the number of underutilized nodes is above the configured value. This can be helpful in large clusters where a few nodes might be underutilized frequently or for a short period of time.
+
+Duplicate Pods::
+The `duplicates` strategy ensures that there is only one Pod associated with a ReplicaSet, ReplicationController, Deployment, or Job running on same node. If there are more, then those duplicate Pods are evicted for better spreading of Pods in a cluster.
++
+This situation could occur after a node failure, when a Pod is moved to another node, leading to more than one Pod associated with a ReplicaSet, ReplicationController, Deployment, or Job on that node. After the failed node is ready again, this strategy evicts the duplicate Pod.
+
+Violation of inter-pod anti-affinity::
```

Comment:
This (and the others) can become confusing, because we have different names for the strategies in our descheduler operator than what the actual names are in the descheduler. I think both need to be documented, but I opened an issue in the descheduler operator so that we can support the actual strategy names (https://github.com/openshift/cluster-kube-descheduler-operator/issues/83).

Reply:
And a PR to fix that: https://github.com/openshift/cluster-kube-descheduler-operator/pull/95

Reply:
@damemi Is this change making it for 4.4? If so, I'll go ahead and make these updates to only use the actual names (unless there's a good reason why the short names need to be mentioned). It'd be better to avoid confusion if we can.

Reply:
The PR merged and I saw that a new image was built (https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=1129462). So I think this made it into 4.4, yes

Reply:
Thanks, I've made the updates.

Reply:
Looks good to me